doc/features.rst

```diff
@@ -17,7 +17,7 @@ ________
   
 * Supports a truly reproducible workflow by minimizing the code needed for analysis and visualization.
 * Already used in a variety of research projects, from conception to final publication.
-* All HoloViews objects can be pickled and unpickled.
+* All HoloViews objects can be pickled and unpickled, with no plotting-library dependencies.
```

Comment:
That's a really good point. Maybe our ``hv.Store.load`` could somehow have a flag to toggle loading of the style options, afaik it will simply skip them if they can't be loaded already.